test/plugin/test_out_elasticsearch_dynamic.rb

```diff
@@ -474,13 +474,33 @@ def test_adds_logstash_timestamp_when_configured
     driver.configure("logstash_format true\n")
     stub_elastic_ping
     stub_elastic
-    ts = DateTime.now
-    time = Fluent::EventTime.from_time(ts.to_time)
+    begin
+      time = Fluent::EventTime.new(Time.now.to_i, 000000000)
```

Comment:
This plugin already depends on Fluentd v0.14 or later.
`Fluent::EventTime.new` always exists.